lib/audited/auditor.rb

```diff
@@ -336,6 +336,7 @@ def comment_required_state?
 
       def combine_audits_if_needed
         max_audits = audited_options[:max_audits]
+        max_audits = Integer(method(max_audits).call).abs if max_audits.is_a?(Symbol)
```

Comment:
Instead of `is_a?(Symbol)` check more directly `respond_to?(max_audits)`. This checks if the value of `max_audits` is a method in the calling context